docs/pipe-first.md

```diff
@@ -2,4 +2,41 @@
 title: Pipe First
 ---
 
-This section has been moved to the new ReScript project: https://rescript-lang.org/docs/manual/v8.0.0/pipe. More info on the transition [here](https://rescript-lang.org/blog/bucklescript-is-rebranding).
+Pipe first is the operator to apply a function to a value where data is passed as the first argument. `->` is a convenient operator that allows you to "flip" your code inside-out. `a(b)` becomes `b->a`.
+
+```reason
+let result = value->function;
+```
+
+Imagine you have the following:
+
+```reason
+validateAge(getAge(parseData(person)))
+```
+
+This is slightly hard to read, since you need to read the code from the innermost part, to the outer parts. Use Pipe First to streamline it
+
+```reason
+person
+  ->parseData
+  ->getAge
+  ->validateAge
+```
+
+Basically, `parseData(person)` is transformed into `person->parseData`, and `getAge(person->parseData)` is transformed into `person->parseData->getAge`, etc.
+
+**This works when the function takes more than one argument too**.
+
+```reason
+a(one, two, three)
+```
+
+is the same as
+
+```reason
+one->a(two, three)
+```
+
+This works with labeled arguments too.
```

Comment:
This statement could use more explanation. Standing alone like can be confusing 

Maybe an example too?